cli/command/image/history.go

```diff
@@ -44,12 +47,28 @@ func NewHistoryCommand(dockerCli command.Cli) *cobra.Command {
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Only show image IDs")
 	flags.BoolVar(&opts.noTrunc, "no-trunc", false, "Don't truncate output")
 	flags.StringVar(&opts.format, "format", "", flagsHelper.FormatHelp)
+	flags.StringVar(&opts.platform, "platform", "",
+		`Specify a platform from a multi-platform image to show the history for.
+If the platform is not specified, the host platform is preferred if it's available, otherwise any available platform is used.
+
+Format: os[/arch[/variant]]
+Example: "linux/amd64"`)
```

Comment:
How about including the full cmd?

```suggestion
Example: "docker image history --platform linux/amd64"`)
```

Reply:
I (somewhat) assume that this was done to allow defining a global `const` for these flags and for them to be shared (similar to what we did for `--format` 🤔), but I need to check if they differ in other ways